source/material_model/visco_plastic.cc

```diff
@@ -198,63 +193,44 @@ namespace aspect
               if (use_viscous_strain_weakening == true)
                 strain_ii = composition[this->introspection().compositional_index_for_name("viscous_strain")];
 
-              viscous_weakening = calculate_viscous_weakening(strain_ii, j);
+              // Apply strain weakening of the viscous viscosity
+              viscosity_pre_yield *= calculate_viscous_weakening(strain_ii, j);
```

Comment:
Is this term is going to be moved into the "if" statement above in a later a pull request? That's what I recall, but wanted to double check.

Reply:
Yes. For this pull request I didn't want to change any functionality, because I change lots of lines, so I want to make sure that none of the tests are changed and everything still works. Once we merge this, I have a follow-up pull request ready that actually fixes this.